test/dlc_tests/conftest.py

```diff
@@ -291,15 +305,211 @@ def ec2_instance_role_name(request):
 
 
 @pytest.fixture(scope="function")
-def ec2_instance_ami(request):
-    return request.param if hasattr(request, "param") else UBUNTU_18_BASE_DLAMI_US_WEST_2
+def ec2_instance_ami(request, region):
+    return (
+        request.param
+        if hasattr(request, "param")
+        else UBUNTU_18_BASE_DLAMI_US_EAST_1
```

Comment:
We should switch to Ubuntu20 now, maybe in other PR

Reply:
Agreed. Let's do this in a separate PR.